osquery/tables/system/linux/acpi_tables.cpp

```diff
@@ -48,8 +49,8 @@ void genACPITable(const std::string& table, QueryData& results) {
     r["size"] = INTEGER(-1);
   } else {
     r["size"] = INTEGER(table_content.size());
-    r["md5"] = osquery::hashFromBuffer(
-        HASH_TYPE_MD5, table_content.c_str(), table_content.length());
+    r["sha1"] = hashFromBuffer(
```

Comment:
It could be left as MD5. I changed it originally during my attempt to remove OpenSSL from absolutely everywhere, in which case this function (and the similar one in the darwin implementation) would use Boost sha1 instead. This would also make it parallel with `osquery/conversions`

However, I guess I was operating a bit too much on autopilot, and didn't fully consider the implications of backwards compatibility. I'll change it back, since the benefits of the sha1 algorithm (e.g. fewer collisions) don't really matter in this context.